### DISCUSSION TITLE
Slack workspace for community chat

### DISCUSSION THREAD

**Discussion by criskurtin** [just-js/just#41] [Ideas] [answered]
It would be great to have a slack workspace (like Vectorized, KafkaJS, Prisma, etc...) for casual community conversations and various real-time discussions. For example, right now I have a question on the inspector module (how to enable it) - maybe someone in the community can help and answer immediately, or relatively soon. If the question proves relevant for greater amount of people, it can be placed here, too. Or in the wiki.

**Comment by paulocoghi** [comment, marked as answer]
There is a Gitter community available! https://gitter.im/just-js/community

IMHO, I believe gitter is easier to participate since it uses Github accounts to login, it doesn't require invites, and also makes conversations publicly available.
